app/database/check_and_create_db.py

```diff
@@ -0,0 +1,64 @@
+import sys
+
+from sqlalchemy import text, create_engine, Engine
+
+
+def check_and_create_database(DB_USER: str, DB_PASS: str, DB_HOST: str, DB_PORT: str, DB_NAME: str) -> Engine:
```

Comment:
This is a normal function and the naming convention for parameters is not to use UPPER_CASE_SNAKE_NOTATION like this, but to use lower_case_snake_notation like this. No matter how the function is called, this is still a convention even if arguments will be extracted from environment variables. You can use env variables directly, without passing them through the arguments, which is a better option, especially because those env variables are used only here when the session is created. There is no need for them in the caller function. (in your case, you don't need them in database.py except for passing to this function. Then there is no need to extract them there just to pass them here since no validation or something else is done in the caller function)